src/Main.java

```diff
@@ -2,14 +2,86 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        //TIP Press <shortcut actionId="ShowIntentionActions"/> with your caret at the highlighted text
-        // to see how IntelliJ IDEA suggests fixing it.
-        System.out.printf("Hello and welcome!");
-
-        for (int i = 1; i <= 5; i++) {
-            //TIP Press <shortcut actionId="Debug"/> to start debugging your code. We have set one <icon src="AllIcons.Debugger.Db_set_breakpoint"/> breakpoint
-            // for you, but you can always add more by pressing <shortcut actionId="ToggleLineBreakpoint"/>.
-            System.out.println("i = " + i);
+        int clientOS = 0;
+        if (clientOS == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке");
+        }
+        else if (clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        int year = 2014;
+        if (year < 2015 && clientOS == 0) {
+            System.out.println("Установите облегченную версию приложения для iOS по ссылке");
+        }
+        else if (year < 2015 && clientOS == 1) {
+            System.out.println("Установите облегченную версию приложения для Android по ссылке");
+        }
+        else if (year >= 2015 && clientOS == 0) {
+            System.out.println("Установите версию приложения для IOS по ссылке");
+        }
+        else if (year >= 2015 && clientOS == 1) {
+            System.out.println("Установите версию приложения для Android по ссылке");
+        }
+
+        int year2 = 2000;
```

Comment:
```suggestion
        year = 2000;
```
можно было не заводить новую переменную, а просто переиспользовать уже созданную. К тому же названия переменных с цифрами на конце - это в 99% случаев название, которое не добавит ясности